README.md

```diff
@@ -190,6 +194,29 @@ If you just want to experiment with Jenkins X, you can set force_destroy to true
 
 :warning: **Note**: If you set `force_destroy` to false, and run a `terraform destroy`, it will fail. In that case empty the s3 buckets from the aws s3 console, and re run `terraform destroy`.
 
+### EKS node groups
+This module provisions self-managed worker nodes by default.
+In order to provision EKS node groups (EKS managed worker groups) create a _main.tf_ with the following content:
+```terraform
+module "eks-jx" {
+  source  = "jenkins-x/eks-jx/aws"
+  enable_node_group   = true
+  enable_worker_group = false
+}
+
+output "vault_user_id" {
```

Comment:
on a completely unrelated note, it would be nice to have another check whether we finally could get rid of this. See #20

Reply:
ok, I will check and post here.